traveler-ios-ui/TravelerKitUI/ModelExtensions/CatalogItemDetails+TravelerUI.swift

```diff
@@ -15,9 +15,16 @@ extension CatalogItemDetails {
             return nil
         }
 
-        let attr = try? NSMutableAttributedString(data: description.data(using: .utf8)!,
+        guard let attr = try? NSMutableAttributedString(data: description.data(using: .utf8)!,
                                                   options: [.documentType : NSAttributedString.DocumentType.html],
-                                                  documentAttributes: nil)
+                                                  documentAttributes: nil) else {
+            return nil
+        }
+        
+        if #available(iOS 13.0, *) {
+            // This is to maintain the label color when using attributed strings in dark mode
+            attr.addAttribute(.foregroundColor, value: UIColor.label, range: NSRange(location: 0, length: attr.string.count))
```

Comment:
why the need of the guard let? if Im not mistaken you can just do attr?.addAttribute(.foregroundColor....)
attr doesn't need have a value to be returned

Reply:
It's to retrieve the text's length for the range.  No need to add the attribute at all if the attributed text is nil.

Reply:
Removed, had it for the length retrieval, but realized I could just force unwrap safely since it's an optional chaining.